### PR TITLE
Solved multiply.js coverage and add prime library

### PR DESCRIPTION
**Pull request by cartovarc**

### CODE REVIEW

File: src/prime.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  return true;
+}
+
+function getNthPrime(n) {
+  if (!isInteger(n)) throw new Error('n must be an integer');
+  if (n <= 0) throw new Error('n must be greater than 0');
+  if (n === 1) return 2;
+  var countPrimes = 1;
+  var imax = 2000000;
+  for (var i = 3; i < imax; i = i + 2) {
```
Comment (lpatiny) [inline src/prime.js]: We should not use `var` anymore. If a variable never change you should use `const` or in this case use `let`.

Comment (lpatiny) [reply src/prime.js]: The problem with `var` is : https://www.w3schools.com/js/js_hoisting.asp

**Review by lpatiny — COMMENTED** [review]
Cool ! Just a small information about var / let / const.

File: src/prime.js (hunk 1 above)
Comment (lpatiny) [reply src/prime.js]: Maybe you could improve the code to have:
- JsDoc (nice to document the code ;)
- Add an option (an object with default value) to define the max iteration.

Comment (cartovarc) [reply src/prime.js]: Thanks! We will consider it.  We will make a new pull request with these problems solved

**Pull request closed without merging**